pkg/util/logging/default_logger.go

```diff
@@ -31,11 +34,31 @@ func DefaultHooks() []logrus.Hook {
 	}
 }
 
+type LogFormatOptions struct {
+	LogFormat    *flag.Enum
```

Comment:
You can just embed the `*flag.Enum` (so remove `LogFormat` as the field name) -- similarly to https://github.com/heptio/velero/blob/master/pkg/util/logging/log_level_flag.go#L32